cmd/root.go

```diff
@@ -45,24 +45,7 @@ func NewRootCmd() *cobra.Command {
 	rootCmd.AddCommand(newUpgradeCmd())
 	rootCmd.AddCommand(newScaleCmd())
 	rootCmd.AddCommand(newDcosUpgradeCmd())
-
-	var completionCmd = &cobra.Command{
```

Comment:
moved this to a function, fyi @stuartleeks @jackfrancis

Reply:
Cool :-)